salt/modules/pcs.py

```diff
@@ -208,14 +250,42 @@ def cluster_setup(nodes, pcsclustername="pcscluster", extra_args=None):
     """
     cmd = ["pcs", "cluster", "setup"]
 
-    cmd += ["--name", pcsclustername]
+    if __use_new_commands():
+        #cmd += [pcsclustername, "--force"]
+        cmd += [pcsclustername]
+    else:
+        #cmd += ["--name", pcsclustername, "--force"]
+        cmd += ["--name", pcsclustername]
```

Comment:
I've fixed the process so that it isn't necessary anymore. It can still be passed in extraargs but doesn't necessarily always need to be there anymore.

Reply:
k, can you remove the comments on line 252 and 249 then.